### PR TITLE
0.18 fix Material

### PR DESCRIPTION
**Pull request by berndhahnebach**
- [x] Branch rebased on latest master `git pull --rebase upstream master`
- [x] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)

on my last PR for FEM I wrote this will be my last for FEM. Here is the last PR for 0.18 for Material from my side. Hopefully I am not to late. No translations changes in the PR. Mainly material cards. Only very rare changes in the card writer code.

Bernd

FreeCAD 0.18 rules :)

### CODE REVIEW

**Comment by berndhahnebach** [comment]
just forced pushed some commit, but I am done finally.

File: src/Mod/Material/StandardMaterial/ABS-Generic.FCMat [hunk 1]
```diff
@@ -1,19 +1,26 @@
 ; ABS-Generic
 ; (c) 2013 Juergen Riegel (CC-BY 3.0)
-; information about the content of such cards you can find here:
-; http://www.freecadweb.org/wiki/index.php?title=Material
+; information about the content of such cards can be found on the wiki:
+; https://www.freecadweb.org/wiki/index.php?title=Material
```
Comment (luzpaz) [inline src/Mod/Material/StandardMaterial/ABS-Generic.FCMat]: Why not https://www.freecadweb.org/wiki/Material ?

Comment (berndhahnebach) [reply src/Mod/Material/StandardMaterial/ABS-Generic.FCMat]: done

**Comment by wwmayer** [comment]
Merged.

**Pull request closed without merging**